scylla-cdc/src/stream_generations.rs

```diff
@@ -0,0 +1,540 @@
+use futures::stream::StreamExt;
+use scylla::batch::Consistency;
+use scylla::frame::response::result::Row;
+use scylla::frame::value::Timestamp;
+use scylla::query::Query;
+use scylla::{IntoTypedRows, Session};
+use std::error::Error;
+use std::sync::Arc;
+use uuid::Uuid;
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Ord, PartialOrd)]
+pub struct Generation(chrono::Duration);
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Ord, PartialOrd)]
+pub struct StreamID(Vec<u8>);
```

Comment:
Can we turn this into
```rust
#[derive(Debug, Clone, Eq, PartialEq, Hash, Ord, PartialOrd, FromRow)]
pub struct StreamID{id: Vec<u8>}

impl Value for StreamID {
    fn serialize(&self, buf: &mut Vec<u8>) -> Result<(), ValueTooBig> {
        self.id.serialize(buf)
    }
}
```
this? It would make using this type in other modules easier.